packages/react/src/components/Avatar/Avatar.tsx

```diff
@@ -23,10 +24,10 @@ export interface AvatarProps extends UIComponentProps {
   name?: string
 
   /** Size multiplier. */
-  size?: number
+  size?: SizeValue
```

Comment:
Can you check if there are some usages in the prototypes (I am pretty sure there is employee card prototype that has bigger Avatar)

Reply:
sure, thanks for heads up!